### DISCUSSION TITLE
Gauge Colouring Option - Colour from Zero, instead of Min. Value

### DISCUSSION THREAD

**Discussion by headbouyJB** [grafana/grafana#38234] [Dashboards, Panels & Visualizations]
**What would you like to be added**:
I would like an option to have the 'Gauge' panel coloured from the 'Zero' point, painting to the left for negative values and to the right for positive values please! 
The current gauge always paints upwards from the 'Min' value, which seems arbitrary when using both positive and negative values.

**Why is this needed**:
My own use case is for electricity monitoring, with a gauge showing the grid import/export figure. 
Right now, the gauge paints all the way from my -10kW minimum, right up to +1kW or whatever I'm using.
I run into the same issue with battery charge/discharge rates.

Colouring the gauge from 'Zero' gives an indication of flow direction at a quick glance, as well as the size of that flow.

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by koen-lee** [comment]
Duplicate of https://github.com/grafana/grafana/discussions/38273 I think.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
